source/retryable-reads/tests/unified/ExceededTimeLimit.yml

```diff
@@ -0,0 +1,76 @@
+description: "ExceededTimeLimit is a retryable read"
+
+schemaVersion: "1.3"
+
+runOnRequirements:
+  - minServerVersion: "4.2"
+    topologies: [replicaset, sharded, load-balanced]
+    auth: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents: [ commandStartedEvent ]
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name find-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+      - { _id: 4, x: 44 }
+      - { _id: 5, x: 55 }
+      - { _id: 6, x: 66 }
+
+tests:
+  - description: "Find succeeds on second attempt after ExceededTimeLimit"
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: *client0
+          failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+              failCommands: [ "find" ]
+              closeConnection: false
```

Comment:
```suggestion
```

Appears not needed. `closeConnection` [defaults to false](https://github.com/mongodb/mongo/wiki/The-%22failCommand%22-fail-point#failcommand-parameters).